loadtest/user/userentity/websocket.go

```diff
@@ -18,6 +21,46 @@ const (
 	maxWebsocketFails             = 7
 )
 
+func (ue *UserEntity) handleReactionEvent(ev *model.WebSocketEvent) error {
+	var data string
+	if el, ok := ev.Data["reaction"]; !ok {
+		return fmt.Errorf("reaction data is missing")
+	} else if data, ok = el.(string); !ok {
+		return fmt.Errorf("reaction data not of type string")
+	}
+
+	var reaction *model.Reaction
+	if err := json.Unmarshal([]byte(data), &reaction); err != nil {
```

Comment:
Isn't `reaction` already a pointer?
```suggestion
	if err := json.Unmarshal([]byte(data), reaction); err != nil {
```

Reply:
Correct, but `json.Unmarshal` will fail if the passed pointer is `nil`. So either we allocate the reaction before (and then pass a pointer to it) or we need to actually pass a pointer to a pointer.